packages/host/app/components/operator-mode/container.gts

```diff
@@ -57,6 +62,7 @@ interface Signature {
 export interface OperatorModeState {
```

Comment:
just a general observation--why does this interface live in this component when it is only ever used in the operator-mode-state-service? probably that is where it should live...